packs/pc-dlc.pack.json

```diff
@@ -447,11 +447,11 @@
         },
         "extension/scorpion-robo/data/maps/heat/lab/room-final.json/entities/123/settings/condAnims/0/interact/event/6/thenStep/0/text": {
                 "orig": "Interested \\c[3]\\v[lore.title.seeker]s\\c[0] should know that \\c[3]each of their kind\\c[0] is destined to enter \\c[3]only one of the 4 facilities\\c[0] and find a part of the information.",
-                "text": "Les \\c[3]\\v[lore.title.seeker]s\\c[0] intéressés doivent savoir que \\c[3]chaque type d'entre eux\\c[0] est destiné qu'à entrer que dans \\c[3]une seule de ces 4 installations\\c[0] pour trouver une partie de l'information."
+                "text": "Les \\c[3]\\v[lore.title.seeker]s\\c[0] intéressés doivent savoir que \\c[3]chaque classe\\c[0] n'est destinée qu'à entrer que dans \\c[3]une seule de ces 4 installations\\c[0] pour trouver une partie de l'information."
```

Comment:
It's not related to classes, basically each of the four platform (pc, switch, xbox, ps4) have a similar quest with a different hint at the end, which must be shared to deduce the location to a final lab.